index.src.html

```diff
@@ -448,13 +450,19 @@
   <pre class="idl">
     dictionary CredentialRequestOptions {
       CredentialMediationRequirement mediation = "optional";
+      AbortSignal signal;
     };
   </pre>
   <div dfn-for="CredentialRequestOptions" dfn-type="dict-member">
     :   <dfn>mediation</dfn>
     ::  This property specifies the mediation requirements for a given credential request. The
         meaning of each enum value is described below in {{CredentialMediationRequirement}}.
         Processing details are defined in [[#algorithm-request]].
+    :   <dfn>signal</dfn>
+    ::  This property lets the developer abort an ongoing 
+        {{[[DiscoverFromExternalSource]](options)}} operation and subequently all related 
+        [=authenticatorGetAssertion=] operations. Aborted promise would 
```

Comment:
Sure. But I'd prefer calling out AuthenticatorMakeCredential because of the same logic I mentioned above. I'd prefer this PR to just webauthn and handle abort for Credman overall in another PR. I do appreciate the added detail about how an aborted operation may complete.